tests/acceptance/percysnapshot.js

```diff
@@ -0,0 +1,63 @@
+import { setupServer, shutdownServer } from './server';
+import percySnapshot from '@percy/testcafe';
+import UniversalPage from './pageobjects/universalpage';
+import VerticalPage from './pageobjects/verticalpage';
+import FacetsPage from './pageobjects/facetspage';
+import { Selector } from 'testcafe';
+
+/**
+ * This file contains testcafe tests that take percy snapshots.
+ * These have to be separated out because Percy only supports
+ * chrome and firefox.
+ */
+fixture`Universal search page works as expected`
+  .before(setupServer)
+  .after(shutdownServer);
+
+test.page`http://localhost:9999/tests/acceptance/fixtures/html/universal`(
+  'Universal page',
+  async t => {
+    await percySnapshot(t, `universal page pre search`);
+
+    const searchComponent = UniversalPage.getSearchComponent();
+    await searchComponent.enterQuery('Tom');
+    await searchComponent.submitQuery();
+    await t.expect(Selector('.yxt-Results').exists).ok();
+
+    await percySnapshot(t, `universal page post search`);
```

Comment:
Does this PR allow for different snapshots per minor? If not, I think we need to figure that bit out. Otherwise the tests are likely to get turned off or ignored. Minors may have different functionality. A later minor may have UI/UX that an earlier doesn't. If we tried to hold both minors to the same snapshot, we could run into trouble.

Reply:
originally, I assumed Percy was "dumb" and all it did was look at the name of the snapshot, and compare it to the most recent snapshot with the same name. In reality Percy claims it has smart logic for choosing the base branch to compare to, and will try to pick the most recent commit on the branch (I did some light testing and this seems to be the case?) . (https://docs.percy.io/docs/baseline-picking-logic) So new prs on a support/v1.7 will be compared to the last commit on support/v1.7, and won't be compared to say the develop branch

Reply:
update: percy is not that smart, and looks like it has to be used with github PR integration for more our git flow. circleci doesn't let you run a build only when a pull request has been made, but github actions DOES let you do that, so I've updated this pr to use github actions for percy, instead.